astropy/utils/data.py

```diff
@@ -871,18 +871,31 @@ def compute_hash(localfn):
     return h.hexdigest()
 
 
-def _find_pkg_data_path(data_name, package=None):
-    """
-    Look for data in the source-included data directories and return the
-    path.
-    """
+def get_pkg_data_path(*path, package=None):
+    """Make path from source-included data directories.
```

Comment:
"Make" is misleading as this function should not be creating new directories.
```suggestion
    """Get path from source-included data directories.
```